README.md

```diff
@@ -1,6 +1,6 @@
 # `sgeop`: Street Geometry Processing Toolkit
 
-[![Continuous Integration](https://github.com/uscuni/sgeop/actions/workflows/testing.yml/badge.svg)](https://github.com/uscuni/sgeop/actions/workflows/testing.yml)
+[![Continuous Integration](https://github.com/uscuni/sgeop/actions/workflows/testing.yml/badge.svg)](https://github.com/uscuni/sgeop/actions/workflows/testing.yml) [![codecov](https://codecov.io/gh/uscuni/sgeop/branch/main/graph/badge.svg?token=VNn0WR5JWT)](https://codecov.io/gh/uscuni/sgeop)
```

Comment:
I have requested access to configure `codecov.io` for the `uscuni/sgeop` repo. Not sure if a notification was sent to you or not @martinfleis ?

Reply:
approved

Reply:
Still nothing showing up. 99% sure it's because I am not listed as a member of the [`uscuni`](https://github.com/uscuni) organization (and that's why we couldn't work it out for `simplification` either). Can you add me there and we'll see if that takes care of it?

Reply:
![Screenshot 2024-10-06 at 5 01 28 PM](https://github.com/user-attachments/assets/3879bc1f-df5b-413d-bcdf-2acdbf87f55f)

Reply:
I think this is done. If not, please send me the link where I need to click something.